contracts/Removal.sol

```diff
@@ -118,6 +149,15 @@ contract Removal is
     }
   }
 
+  function mintBatch(
+    address,
+    uint256[] calldata,
+    uint256[] calldata,
+    bytes calldata
+  ) public pure override {
+    revert("Removal: mintBatch standard ERC1155 disabled");
```

Comment:
should probably disable `mint` as well

Some part of me wants to just use the bytes data instead of adding a new function and reverting here though. Thoughts?

Reply:
I also wanted to just use the bytes data to do what we needed here, but the issue is that if you're decoding bytes, you have to tell the decoder exactly what to parse out of it.  meaning you can't really pass a variable-length piece of a data each time like you can with the dynamic array params.  This is a problem because we want to be able to batch mint an arbitrary number of removals each time...  However, if we were comfortable saying that when you mint a batch of removals they all have to have the same escrow schedule startTime (a reasonable scenario since we're probably usually minting removals for a single project at a time, and those removals probably all share an NRT agreement date), then we could put a single value for the escrow schedule startTime in the bytes data and apply that to the whole batch.  

Since this is not a contract that really needs to conform to an interface (since we're the only ones interacting with it) then I'm not sure if there are really any important concerns about what this function signature looks like.